explore/src/main/scala/explore/config/ConfigurationTile.scala

```diff
@@ -192,6 +292,29 @@ object ConfigurationTile:
         .render { (props, ctx) =>
           import ctx.given
 
+          val posAngleConstraintAligner
+            : Aligner[PosAngleConstraint, Input[PosAngleConstraintInput]] =
+            Aligner(
+              props.posAngle,
+              UpdateObservationsInput(
+                WHERE = props.obsId.toWhereObservation.assign,
+                SET = ObservationPropertiesInput()
+              ),
+              ObsQueriesGQL
+                .UpdateObservationMutation[IO]
+                .execute(_)
+                .void
+                .switching(props.agsState.async, AgsState.Saving, AgsState.Idle)
```

Comment:
@cquiroz I'm not sure exactly what the `agsState` is for. Should it be `switched` when updating the configuration, too?

Reply:
`agsState` is used to show where we are in the calculation process, loading stars, calculating, etc
I think it should recalculate if conf changes

Reply:
So, should it be set/unset for any configuration changes?  But, doesn't the `switching` here set it back to `Idle` when we get the response back from the API mutation? That seems it wouldn't have anything to do with the actual AGS calculation cycle.

Reply:
It seems that the "calculating" gets displayed for a PAC change, even without this switching here. So, I'll remove it.